scene/resources/shader.cpp

```diff
@@ -121,6 +126,11 @@ void Shader::get_shader_uniform_list(List<PropertyInfo> *p_params, bool p_get_gr
 	List<PropertyInfo> local;
 	RenderingServer::get_singleton()->get_shader_parameter_list(shader, &local);
 
+#ifdef TOOLS_ENABLED
+	DocData::ClassDoc class_doc = DocData::ClassDoc();
```

Comment:
```suggestion
	DocData::ClassDoc class_doc;
```
Unnecessary default value